content/docs/get-started-with-neon/query-with-psql-editor.md

```diff
@@ -1,20 +1,124 @@
+
+After making your selection, you are directed to check the terminal. In the terminal, you are connected to the project, and  connection information similar to the following is displayed:
+
+```bash
+NOTICE:  Connecting to database.
+psql (14.5 (Ubuntu 14.5-0ubuntu0.22.04.1))
+Type "help" for help.
+
+user1=>
+```
+
+**_Note_**: When using _`psql` quick auth_ to connect, the `psql` prompt shows your local terminal user name instead of the database name that is shown for the other `psql` connection methods described in this topic. However, you are logged in to the default `main` database as the Neon `web_access` user, which you can verify by running this query:
+
+```bash
+user1=> SELECT current_user;
+ current_user 
+--------------
+ web_access
+(1 row)
+
+user1=> SELECT current_database();
+ current_database 
+------------------
+ main
+ (1 row)
+```
+
+## Connecting with an exported password
+
+**_Warning_**: Some operating systems allow non-root users to view process environment variables when using the `ps` command. For security reasons, consider using a password file in such cases. 
+
+To connect with an exported password:
+
+1. In your terminal, export the database user's password to the `PGPASSWORD` environment variable; for example:
+
+```bash
+export PGPASSWORD=En5v0dJoVpRL
+```
+
+The database user's password was provided to you when you created the project.
+
+2. Connect with the following command:
+
+```bash
+psql postgres://<user>:$PGPASSWORD@<project_ID>.cloud.neon.tech:5432/main
+```
+
+where:
+
+- `<user>` is the database user, which can be found on the Neon console **Dashboard** tab, under **Connection Details**. 
+- `<project_ID>` is the Neon project ID, which can be found on the Neon console **Settings** tab, under **General Settings**.
+
+## Connecting with a password saved to a password file
+
+To connect with a password saved to a `.pgpass` password file:
+
+1. In your terminal, run the following commands to create and configure the `.pgpass` file: 
+
+```bash
+touch ~/.pgpass && \
+chmod 0600 ~/.pgpass && \
+echo -e "<project_ID>.cloud.neon.tech:5432:main:<user>:<password>\n$(cat ~/.pgpass)" >> ~/.pgpass
+```
+
+2. Connect with the following command:
 
 ```bash
-psql -h postgres://<username>@<project_id>.cloud.neon.tech main
+psql -h <project_ID>.cloud.neon.tech -U <user> main
 ```
 
-Run a simple query:
+where:
+
+- `<project_ID>` is the ID of the Neon project, which can be found on the Neon console **Settings** tab, under **General Settings**.
+- `<password>` is the database user's password, which was provided to you when you created the project
+- `<user>` is the database user, which can be found on the Neon console **Dashboard** tab, under **Connection Details**.
+
+## Running queries
+
+After establishing a connection, try running the following queries:
 
 ```sql
-create my_table as select now();
-select * from my_table;
+main=> CREATE TABLE my_table AS SELECT now();
```

Comment:
I think it's better to remove the "main=>", the user might copy/paste the commands to their terminal

Reply:
Good point! Fixed.